Unix/http/httpclientauth.c

```diff
@@ -1465,13 +1563,16 @@ HttpClient_EncryptData(_In_ HttpClient_SR_SocketData * handler, _Out_ Page **pHe
     {
          *pdst = *psrc;
     }
+
     memcpy(pdst, MULTIPART_ENCRYPTED, MULTIPART_ENCRYPTED_LEN);
     pdst += MULTIPART_ENCRYPTED_LEN;
     memcpy(pdst, original_host_header, host_header_len);
     pdst += host_header_len; 
-    *pdst++ ='\r';
-    *pdst++ ='\n';
+    *pdst++ = '\r';
+    *pdst++ = '\n';
 
+
+//    PAL_Free(*pHeader); We hav no way of knowing how the pHeader got there
```

Comment:
Commented out. Not sure what you're saying here. Is the header leaked? Is the header not leaked? What are you intending to do with pHeader?